core/Pos/Core/Arbitrary.hs

```diff
@@ -119,6 +121,12 @@ instance Arbitrary Types.EpochOrSlot where
         ]
     shrink = genericShrink
 
+-- | TODO: Replace with code from Pos.Binary.Cbo.Test:arbitraryUnparsedFields
+-- after transition to CBOR serialization.
+instance Arbitrary UnparsedFields where
```

Comment:
@arybczak was rightly pointing out here that we should override this `Arbitrary` instance with the one he defined in that module (now called `Test.Pos.CborSpec`). I will update the stale comment!